101-backup-la-reporting/prereqs/prereq.azuredeploy.json

```diff
@@ -3,37 +3,44 @@
   "contentVersion": "1.0.0.0",
   "parameters": {
     "workspaceName": {
-      "type": "string",
-      "metadata": {
-        "description": "Specify the workspace name"
-      }
+      "type": "string"
     },
-    "workspaceLocation": {
-      "type": "string",
-      "metadata": {
-        "description": "Specify the workspace region"
-      },
-      "defaultValue": "West Europe"
+    "location": {
+      "type": "string"
     }
   },
   "variables": {},
   "resources": [
     {
+      "type": "Microsoft.Resources/deployments",
+      "apiVersion": "2019-05-01",
       "name": "[parameters('workspaceName')]",
-      "type": "Microsoft.OperationalInsights/workspaces",
-      "apiVersion": "2015-11-01-preview",
-      "location": "[parameters('workspaceLocation')]"
-      
+      "resourceGroup": "[resourceGroup().name]",
+      "properties": {
+        "mode": "Incremental",
+        "template": {
+          "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+          "contentVersion": "1.0.0.0",
+          "resources": [
+            {
+              "name": "[parameters('workspaceName')]",
+              "type": "Microsoft.OperationalInsights/workspaces",
+              "apiVersion": "2015-11-01-preview",
```

Comment:
/providers API shows the following versions available for workspaces:
            "apiVersions": [
                "2017-04-26-preview",
                "2017-03-15-preview",
                "2017-03-03-preview",
                "2017-01-01-preview",
                "2015-11-01-preview",
                "2015-03-20"
            ]
        },

Reply:
Updated